README.md

```diff
@@ -60,6 +60,14 @@ Tilix requires the following libraries to be installed in order to run:
 * GSettings
 * [Nautilus-Python](https://wiki.gnome.org/Projects/NautilusPython) (Required for Nautilus integration)
 
+### Optional Fonts
+In the screeshots you'll notice I use powerline shell, in order to ensure this works correctly, you'll need to ensure the fonts are installed and Tilix is aware of your desired font:
```

Comment:
Typo: "screeshots" --> "screenshots"

I would rather not speak with personal pronouns in a README, so maybe better use "In many of the screenshots, the `powerline` statusline shell plugin is used." etc.

Reply:
Good Point, addressed.  Thanks for considering this.